### ISSUE TITLE
Support for alternate services?

### ISSUE THREAD

**Issue by BEEDELLROKEJULIANLOCKHART** [Chesire/Nekome#701]
This is the only recorder of literature and videography that is available via F-Droid that I have located, so although I would be very pleased if this were to become one that is able to record more generic information – not only Japanese – this is definitely too much to request without provision of assistance by myself. Consequently, all that I shall propose is management of alternate trackers of Japanese media, such as “http://anilist.co” and/or “http://myanimelist.net”.

I am currently utilising primarily “http://anilist.co”, so addition of that website would allow me to easily synchronise it with or transfer the data of it to Kitsu, which I have estimated is probably your favourite.

Despite your eventual judgment, I shall be glad if you do consider my proposition.

**Comment by Chesire** [comment]
Thanks for the suggestion, adding support for MAL and Anilist are two things I would love to add, just haven't found the opportunity to. I'll see if I can start working on the anilist api some point soon, although from what I recall its all in GraphQL which I'm not too familiar with.
In the mean time I believe Kitsu and Anilist provide ways to backup/sync to the other service, so might be an idea to try that out

**Comment by rokejulianlockhart** [comment]
@Azmarine13, I agree, although does https://simkl.com support literature? I use it solely for videography.

If so, foes a separate issue for that exist? If I were you, I'd open it.
